### PR TITLE
CA-381075 Fix the Copyright statement for ACK source code

### PR DESCRIPTION
**Pull request by catherineliuchina**

### CODE REVIEW

File: LICENCE [hunk 1]
```diff
@@ -1,6 +1,4 @@
-Copyright (c) 2005-2022 Citrix Systems Inc.
-Copyright (c) 2022-12-01 Cloud Software Group Holdings, Inc.
-All rights reserved.
+Copyright (c) 2005-2023 Cloud Software Group, Inc.
```
Comment (stephenchengCloud) [inline LICENCE:1]: Isn't it better to keep "Copyright (c) 2005-2022 Citrix Systems Inc."? Did you make this change by any official guides?

Comment (catherineliuchina) [reply LICENCE:1]: I followed up https://info.citrite.net/display/xenserver/CSG+XenServer+Rebrand+Requirements to make the change. I will double check whether we need keep "Copyright (c) 2005-2022 Citrix Systems Inc."

**Review by stephenchengCloud — APPROVED** [review]

File: LICENCE [hunk 2]
```diff
@@ -1,6 +1,4 @@
-Copyright (c) 2005-2022 Citrix Systems Inc.
```
Comment (DeliZhangX) [inline LICENCE:1]: These need not to remove as my understanding.

File: acktools/log.py [hunk 3]
```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/python3
 
-# Copyright (c) 2005-2022 Citrix Systems Inc.
```
Comment (DeliZhangX) [inline acktools/log.py:3]: These need not to remove as my understanding.

Comment (catherineliuchina) [reply acktools/log.py:3]: I have recovered the copyright for Citrix

**Review by DeliZhangX — APPROVED** [review]

**Review by stephenchengCloud — APPROVED** [review]

**Pull request merged**